### PR TITLE
Add docs on how to do a future test

### PR DESCRIPTION
**Pull request by scarlehoff**
Addresses #1407

### CODE REVIEW

File: doc/sphinx/source/tutorials/futuretests.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+How to run a Future Test
+========================
+
+The PDF sets generated with the NNPDF methodology are, of course, able to describe the data they were fitted with.
+However, for them to be truly useful, they should be also able to faithfully describe data that the methodology has never seen.
+By checking the :math:`\chi2` of said unseen data we can test the generalization power of the methodology.
+
+Ideally, that would entail constructing a new experiment and obtaining new data we can test the PDF against,
+however, we cannot time travel so we must resort to less robust techniques.
+In NNPDF4 we test the generalization power of the methodology by fitting coherent subsets of data.
```
Comment (RoyStegeman) [inline doc/sphinx/source/tutorials/futuretests.rst:11]:
```suggestion
In NNPDF4.0 we test the generalization power of the methodology by fitting coherent subsets of data.
```

File: doc/sphinx/source/tutorials/futuretests.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+In order to keep consistent with the analogy, in :cite:p:`Cruz-Martinez:2021rgy` we chose Pre-Hera and Pre-LHC as
+those subsets but in reality (and despite the name) any subset of data would be valid.
+The chronological choice, which in practice separates by experiment, ensures that no leakage of information occurs between the in-sample and out-of-sample data
+(for instance, some unknown systematic error that biases data from one given experiment).
+
+Since we are interested in the generalization power of the PDF, we need to take into account the pdf errors when
+computing the :math:`\chi2`, to that end the flag ``use_pdferr`` is provided in ``validphys``.
```
Comment (RoyStegeman) [inline doc/sphinx/source/tutorials/futuretests.rst:18]: Maybe give the explicit expression of chi2 if use_pdferr is true? It's quite easy to understand and removes any ambiguity of what "taking into account the pdf errors" actually means.

Comment (scarlehoff) [reply doc/sphinx/source/tutorials/futuretests.rst:18]: You're right. I'll add it

File: doc/sphinx/source/tutorials/futuretests.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+  meta:
+    title: I didn't change the title
+    keywords: [Guilty]
+    author: Lazy Person
+
+  use_pdferr: True
+
+  future:
+    pdf: {id: future_fit, label: "Future"}
```
Comment (RoyStegeman) [inline doc/sphinx/source/tutorials/futuretests.rst:50]: maybe do?
pdf:
  from_: fit

**Review by RoyStegeman — APPROVED** [review]
Looks good to me

File: doc/sphinx/source/tutorials/futuretests.rst [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  meta:
+    title: I didn't change the title
+    keywords: [Guilty]
+    author: Lazy Person
+
+  use_pdferr: True
+
+  Future:
+    pdf: {id: future_fit, label: "Future"}
+    fit: {id: future_fit, label: "Future"}
```
Comment (scarlehoff) [inline doc/sphinx/source/tutorials/futuretests.rst:51]:
```suggestion
    fit: {id: future_fit, label: "Future"}
    pdf:
      from_: fit
```

File: doc/sphinx/source/tutorials/futuretests.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+    theory:
+      from_: fit
+    theoryid:
+      from_: theory
+
+  Past:
+    pdf: {id: past_fit, label: "Past"}
+    fit: {id: past_fit, label: "Past"}
```
Comment (scarlehoff) [inline doc/sphinx/source/tutorials/futuretests.rst:60]:
```suggestion
    fit: {id: past_fit, label: "Past"}
    pdf:
       from_: fit
```

**Comment by Zaharid** [comment]
FWIW https://docs.nnpdf.science/vp/complex_runcards.html?highlight=capital#multiple-inputs-and-namespaces

**Review by Zaharid — APPROVED** [review]

**Pull request merged**